seqr/urls.py

```diff
@@ -138,13 +140,21 @@
 ]
 
 # NOTE: the actual url will be this with an '/api' prefix
+
+urlpatterns = []
+
+phenotips_urls = '^(?:%s)' % ('|'.join([
```

Comment:
do you mind moving this back to where it was? Its not super important, it just made more sense there